daml/BobTrigger.daml

```diff
@@ -3,24 +3,33 @@
 
 module BobTrigger where
 
--- import DA.Action
 import DA.Foldable
-import DA.Next.Map (Map)
+--import DA.Next.Map (Map)
 import Daml.Trigger
 import Main
 
-rejectTrigger : Trigger () = Trigger with
-  initialize = \_ -> ()
-  updateState = \_ _ () -> ()
-  rule = rejectRule
-  registeredTemplates = AllInDar
-  heartbeat = None
+--data Trigger s = Trigger
+--  { initialize : TriggerInitializeA s
+--  , updateState : Message -> TriggerUpdateA s ()
+--  , rule : Party -> TriggerA s ()
+--  , registeredTemplates : RegisteredTemplates
+--  , heartbeat : Optional RelTime
+--  }
```

Comment:
Any reason to keep commented code around? I generally favour deleting code that is no longer useful.

Reply:
Agreed though still not clear whether this should be used or not. My knowledge of Daml and in particular is negligible and the docs imply one needs this.

Reply:
The docs only show the definition of this type since you need to create a value of this type. You’re not supposed to copy this definition to your own project, it comes from `Daml.Trigger`.